Dynavity/Dynavity/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>529861719898-764hmkpuj78u61c5gl9jbsc26npvs1d6.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.529861719898-764hmkpuj78u61c5gl9jbsc26npvs1d6</string>
+	<key>API_KEY</key>
+	<string>AIzaSyAYHKWULb1fe6Pt2JdRbpzjJk7L5af3zNI</string>
```

Comment:
Is this a security concern?

Reply:
https://stackoverflow.com/questions/44937175/firebase-should-i-add-googleservice-info-plist-to-gitignore

I think it should be fine for now, since for CS3217 purposes the graders will also need access to this file.

If it does become a problem, it shouldn't be too hard to invalidate this and make a new key.